### PR TITLE
Updates for django 1.7.3

### PR DESCRIPTION
**Pull request by ururk**
This was tested on a django 1.7.3 install. Because of one change (mimetype => content_type), this is probably not backwards compatible with earlier versions of django.

### CODE REVIEW

**Comment by ururk** [comment]
This closes #202

File: apps/pages/views.py [hunk 1]
```diff
@@ -116,36 +116,36 @@ def ondemand(req, ensemble_id):
 
 def source(req, n, allow_guest=False):
     source = M.Source.objects.get(pk=n)
-    if source.type==M.Source.TYPE_YOUTUBE: 
-        return __serve_page(req, settings.YOUTUBE_TEMPLATE, allow_guest , mimetype="text/html")
+    if source.type==M.Source.TYPE_YOUTUBE:
+        return __serve_page(req, settings.YOUTUBE_TEMPLATE, allow_guest , content_type="text/html")
```
Comment (Eyas) [inline apps/pages/views.py]: Do you know what Django version content_type was introduced?

Comment (Eyas) [inline apps/pages/views.py]: Never mind; looks like this has been there since 1.5, which should be fine :)

**Comment by karger** [comment]
resolved by 7c0d04f334590446217bd89562c4eab85d527680

**Pull request closed without merging**